src/client/datascience/constants.ts

```diff
@@ -382,7 +382,6 @@ export enum Telemetry {
     GetPreferredKernelPerf = 'DS_INTERNAL.GET_PREFERRED_KERNEL_PERF',
     KernelFinderPerf = 'DS_INTERNAL.KERNEL_FINDER_PERF',
     KernelListingPerf = 'DS_INTERNAL.KERNEL_LISTING_PERF',
-    NativeNotebookKernelSelectionPerf = 'DS_INTERNAL.NATIVE_NOTEBOOK_KERNEL_SELECTION_PERF',
```

Comment:
This is a duplicate, we already have a separate telemetry item for this.
& since this is related to native notebooks not concerned about removing this.